packages/app/src/state/staking/types.ts

```diff
@@ -36,6 +37,7 @@ export type StakingState = {
 	claimableSnxOpRewards: ClaimParams[]
 	selectedEpoch?: number
 	stakingMigrationCompleted: boolean
```

Comment:
Is this still used? As I see the selector got deleted

Reply:
Thanks. Removed.